node/packages/aws-lambda-otel-extension/external/otel-extension-external/index.js

```diff
@@ -268,7 +273,15 @@ module.exports = (async () => {
               case 'eventData':
                 {
                   if (data.record.requestEventPayload) {
-                    sendReport('request', data.record.requestEventPayload);
+                    const { resourceAtt, metricsAtt } = createAttributes(data.record);
+                    sendReport('request', {
+                      ...data.record.requestEventPayload,
+                      timestamp: currentRequestData.logsQueue[0]
+                        ? currentRequestData.logsQueue[0].time
+                        : new Date().getTime() - 100,
```

Comment:
Looking for a way to replace this with the invocation time 🤔

Reply:
Basically I need to make sure the request payload has a timestamp at invocation time (Before any other log timestamps). The request doesn't come with a timestamp and I can't seem to find something reliable on external that will give me a timestamp before any log messages 🤔